jekyll/_cci2/deployment-examples.md

```diff
@@ -1,99 +1,82 @@
 ---
 layout: classic-docs
-title: "Configuring Deploys"
-short-title: "Configuring Deploys"
+title: "Deployment Examples"
+short-title: "Deployment Examples"
 ---
+This document presents example config for a variately of popular deployment targets. Many of the examples use orbs: CircleCI and partners have developed a catalogue of orbs that enable you to quickly deploy applications with minimal config. Details of all orbs can be found in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
 
-CircleCI can be configured to deploy to virtually any service. This document provides instructions and examples for the following platforms:
+**Note**: In order to use orbs you must use `version 2.1` config, and enable pipelines for your project. If you wish to remain using `version 2.0` config, or are using a self-hosted installation of CircleCI Server, the examples shown here are still relevant because you can view the expanded orb source within the [Orbs Registry](https://circleci.com/orbs/registry/) to see how the jobs are built.
```

Comment:
Not sure if this bit about version 2.0 config users looking at the orb source is a good idea but just stuck it in in case.